### PR TITLE
fixed but ugly duped code :)

### PR DESCRIPTION
**Pull request by Srensommer**
wow

### CODE REVIEW

File: src/pages/profile.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { type ChartConfiguration } from "chart.js";
 import Elysia from "elysia";
-import { type Session } from "lucia";
+import { User, type Session } from "lucia";
```
Comment (Faustvii) [inline src/pages/profile.tsx:3]: unused

File: src/pages/profile.tsx [hunk 2]
```diff
@@ -348,13 +348,15 @@ function matchFaceoff(biggestWin: {
   );
 }
 
-function matchOutput({
+async function matchOutput({
```
Comment (Faustvii) [inline src/pages/profile.tsx:351]: async not needed

File: src/pages/profile.tsx [hunk 3]
```diff
@@ -348,13 +348,15 @@ function matchFaceoff(biggestWin: {
...
   match,
   result,
 }: {
   match: MatchWithPlayers;
   result: RESULT;
-}): JSX.Element {
+},
+userId: string,
+): Promise<JSX.Element> {
```
Comment (Faustvii) [inline src/pages/profile.tsx:359]: promise not needed

File: src/pages/profile.tsx [hunk 4]
```diff
@@ -363,17 +365,37 @@ function matchOutput({
...
-        month: "long",
-      })}
-      , <span class="font-bold">{whiteTeam.join(" & ")}</span> faced off against{" "}
-      <span class="font-bold">{blackTeam.join(" & ")}</span> and{" "}
-      {result === RESULT.DRAW ? "tied" : result === RESULT.WIN ? "won" : "lost"}{" "}
-      with {match.scoreDiff} points.
-    </span>
-  );
+
+  const firstSide = MatchStatistics.getPlayersTeam(match, userId)
```
Comment (Faustvii) [inline src/pages/profile.tsx:369]:
```
const userTeam = MatchStatistics.getPlayersTeam(match, userId)
const firstTeam = userTeam === 'black' ?  blackTeam : whiteTeam;
const lastTeam = userTeam === 'black' ? whiteTeam : blackTeam;
```
then you won't have to duplicate the jsx

**Review by Faustvii — CHANGES_REQUESTED** [review]

File: src/pages/profile.tsx (hunk 2 above)
Comment (Srensommer) [reply src/pages/profile.tsx:351]: byebye

File: src/pages/profile.tsx (hunk 3 above)
Comment (Srensommer) [reply src/pages/profile.tsx:359]: byebye

File: src/pages/profile.tsx (hunk 4 above)
Comment (Srensommer) [reply src/pages/profile.tsx]: fixefaxed

**Review by Faustvii — APPROVED** [review]

**Pull request merged**